lib/proxyquire.js

```diff
@@ -132,8 +156,18 @@ Proxyquire.prototype.load = function (request, stubs) {
     }
   }
 
+  this._stubsUsed = {};
```

Comment:
You have to `reset` lookup table on each load. So one have to keep this here.
Also you are not worried about hidden classes in all other situations, so there is no sence to add this field in constructor.